notifier/notifier.go

```diff
@@ -1,30 +1,28 @@
 package notifier
 
 import (
-	"bytes"
 	"context"
-	"encoding/json"
 	"errors"
 	"expvar"
 	"fmt"
 	"log"
-	"net"
 	"net/http"
 	"net/url"
 	"os"
-	"path"
 	"strings"
 	"sync"
 	"time"
 
+	"github.com/skroutz/downloader/backend"
+	h "github.com/skroutz/downloader/backend/http_backend"
+	k "github.com/skroutz/downloader/backend/kafka_backend"
```

Comment:
I think this greatly reduces readability. The default package names are much better IMHO.